Update eventhub_namespace for ZoneRedundant property (#7534)
terraform-providers/terraform-provider-azurerm: website/docs/d/eventhub_namespace.html.markdown

* `maximum_throughput_units` -  Specifies the maximum number of throughput units when Auto Inflate is Enabled.

> * `zone_redundant` - Enabling this property creates a Standard Event Hubs Namespace in regions supported availability zones.

Reviewer: since this is a Data Source (e.g. retrieving a read-only value) we can make this:

```suggestion
* `zone_redundant` - Is this EventHub Namespace deployed across Availability Zones?
```
